javascripts/projectsService.js

```diff
@@ -1,5 +1,6 @@
 /* eslint global-require: "off" */
 /* eslint block-scoped-var: "off" */
+/* eslint prefer-arrow-callback: [ "error" ] */
```

Comment:
Like we've found in #1520, I think we also need to add this, run `npm run prettier-fix` and then commit to make CI pass:


```suggestion
/* eslint prefer-arrow-callback: [ "error" ] */
/* eslint arrow-parens: [ "error", "as-needed" ] */
```

Reply:
this the running one. don't get why its failed.